### ISSUE TITLE
Pass title attribute from button down to rux-icon when specified with rux-button

### ISSUE THREAD

**Issue by QGerrit** [RocketCommunicationsInc/astro#625]
I have a `rux-button` which, due to space requirements, only includes an icon. To give the user a description of what each button does, I set the html `title` attribute on each button, which works if I hover over the part of the `rux-button` which is not displaying the icon, but doesn't work when hovering over the icon itself.

```html
    <rux-button
      icon="palette"
      icon-only
      title="Title text"
    ></rux-button>
```

![image](https://user-images.githubusercontent.com/97202638/183756713-634da3f5-2f05-4489-a601-415a601b0494.png)

![image](https://user-images.githubusercontent.com/97202638/183756851-e352000d-f996-40b4-9176-ad6b88017476.png)

**Comment by micahjones13** [comment]
Hey @QGerrit,

From the context, it seems like you're trying to add a tooltip to the button and icon, correct? If that's the case, then I would recommend staying away from the title attribute. See some reasons [here](https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute). 
Something you've probably noticed is that `rux-icon` actually assigns the title attribute - hence the "palette" showing up. This was an oversight on our part and will be removed in the future, so apologies for the confusion it has caused. 

What I would recommend to create a tooltip is to follow a CSS pattern such as this: https://www.w3.org/WAI/WCAG22/Techniques/css/C7. Also, if you would like a `rux-tooltip` or something similar in the future, our [GitHub discussions](https://github.com/RocketCommunicationsInc/astro/discussions/categories/ideas) are open for any ideas you may have!

Thanks, and please let me know if you have any other questions or need support.

**Comment by jduncanRadBlue** [comment]
The same thing is happening to me.  I have an icon inside a button and when I hover over the icon, the tooltip is the icon's name and not the title which I have specified for the button.  How can I disable the icon's tooltip?

**Comment by QGerrit** [comment]
That's fair, a `rux-tooltip` would be very useful! Having written them by hand before, it's really annoying to do so.

**Comment by micahjones13** [comment]
The `title` attribute on the rux-icon has been removed in the above PR, and will be released with 7.0 since it's considered a breaking change. 
Right now, you can actually change the title attribute of the rux-icon if you use a slot on the button and set the label attribute. Here's an [example sandbox](https://codesandbox.io/s/fervent-bash-g3ycnp?file=/index.html). 
This works because internally we are passing the label into the title on rux-icon (which we shouldn't be doing, and that's why it's being removed). Once the above fix is released, you'll be able to simply set the title on a rux-icon, rather than the label. 

Thanks again for bringing this up and all the feedback it's generated!
